framework/aster-frame/src/bus/pci/cfg_space.rs

```diff
@@ -1,5 +1,7 @@
 // SPDX-License-Identifier: MPL-2.0
 
+//! The PCI configuration space. Reference: [osdev](https://wiki.osdev.org/PCI)
+
```

Comment:
`Reference` should be in a new line. And add an empty line between Reference and the first statement. Other places should be refactored similarly.

```rust
//! The PCI configuration space. 
//! 
//! Reference: [osdev](https://wiki.osdev.org/PCI)
```